src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
+
```

Comment:
4. Error 4: Unused parameter in method
   - The `subtract` method has an unused parameter 'c'.